lib/reply.js

```diff
@@ -485,9 +483,7 @@ function onSendEnd (reply, payload) {
   }
 
   if (reply[kReplyTrailers] === null) {
-    if (!reply[kReplyHeaders]['content-length']) {
-      reply[kReplyHeaders]['content-length'] = '' + Buffer.byteLength(payload)
-    } else if (req.raw.method !== 'HEAD' && reply[kReplyHeaders]['content-length'] !== Buffer.byteLength(payload)) {
+    if (!reply[kReplyHeaders]['content-length'] || (req.raw.method !== 'HEAD' && parseInt(reply[kReplyHeaders]['content-length'], 10) !== Buffer.byteLength(payload))) {
```

Comment:
the line's 164 chars length is less readable than before

Reply:
Is it ok to format it like this?

```js
if (!reply[kReplyHeaders]['content-length'] ||
    (req.raw.method !== 'HEAD' && parseInt(reply[kReplyHeaders]['content-length'], 10) !== Buffer.byteLength(payload))) {
```

or revert it completely?

Reply:
I would go with something:

```
const contentLength = reply[kReplyHeaders]['content-length']
if (!contentLength ||
    (req.raw.method !== 'HEAD' &&
     parseInt(contentLength, 10) !== Buffer.byteLength(payload)
    )
   ) {
```

Reply:
:ok_hand: